src/auth-service/utils/analytics.util.js

```
@@ -2641,6 +2630,759 @@
+      logger.error(`🐛🐛 Internal Server Error ${error.message}`);
+      next(
+        new HttpError(
+          "Internal Server Error",
+          httpStatus.INTERNAL_SERVER_ERROR,
+          { message: error.message }
+        )
+      );
+    }
+  },
+  // New function to get number of new users using Presto
+  getNewUsers: async (request, next) => {
```

Biome (1.9.4)

[error] This property value named getRecentUsers is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named getRecentUsers


(lint/suspicious/noDuplicateObjectKeys)